Makefile

```diff
@@ -75,7 +75,7 @@ AFLAGS = -fsanitize=address #-fsanitize=undefined -fno-omit-frame-pointer
 
 OPT    = -O -g -march=native -ffp-contract=fast -fopenmp-simd
 CFLAGS = -std=c99 $(OPT) -Wall -Wextra -Wno-unused-parameter -fPIC -MMD -MP
-CXXFLAGS = $(OPT) -Wall -Wextra -Wno-unused-parameter -fPIC -MMD -MP
+CXXFLAGS = -std=c++11 $(OPT) -Wall -Wextra -Wno-unused-parameter -fPIC -MMD -MP
```

Comment:
@jedbrown, any strong opinions on specifying C++11 here vs the OCCA portion of the Makefile?

Reply:
I think here is consistent with the `-std=c99` above. Modern compilers default to C11 and C++14 (or their `gnu11`/`gnu14++` variants), and we intend our code to be forward-compatible so all we really care about is the minimum supported feature set.